ansible-advancedlinux/azuredeploy.json

```diff
@@ -0,0 +1,826 @@
+{
+
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+
+	"parameters": {
+		"location": {
+			"type": "string",
+			"defaultValue": "West US",
+			"allowedValues": [
+				"West US",
+				"West Europe",
+				"East US",
+				"East Asia",
+				"Southeast Asia"
+			],
+			"metadata": {
+				"Description": "Region where you want to create all the resources"
+			}
+		},
+		"storageAccountName": {
+			"type": "string",			
+			"metadata": {
+				"Description": "Storage account name pattern.The template will use this value and the resource group name for all the storage accounts."
+			}
+		},
+		"storageAccountType": {
+			"type": "string",
+			"defaultValue": "Standard_LRS",
+			"allowedValues": [
+				"Standard_LRS",
+				"Premium_LRS"
+			],
+			"metadata": {
+				"Description": "Type of storage account. Standard_LRS or Premium_LRS. For Premium Storage, use a DS VM"
+			}
+		},
+		"vmStorageAccountContainerName": {
+			"type": "string",
+			"defaultValue": "cbavhds",
+			"metadata": {
+				"Description": "Container name in which you want to place the vhds"
+			}
+		},
+		"vmNumberOfDataDisks": {
+			"type": "int",
+			"defaultValue": 2,
+			"metadata": {
+				"Description": "Number of data disks (For future versions, currently 2 disks will be created)"
+			}
+		},
+		"vmSizeDataDisks": {
+			"type": "int",
+			"defaultValue": 120,
+			"metadata": {
+				"Description": "Data disks Size in GBs"
+			}
+		},
+		"vmFileSystem": {
```

Comment:
nit: linuxFileSystem as a name? Folks might think this is something specific to the vm on Azure. Probably fine.